modules/vcf2maf/1.0/vcf2maf.smk

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Bruno Grande
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CONFIG`
+CONFIG = config["lcr-modules"]["vcf2maf"]
+LOG = "/logs/" + op._session.launched_fmt
+
+wildcard_constraints:
+    prefix = "[0-9]{2}-.*"
+
+VERSION_UPPER = {
+    "grch37": "GRCh37",
+    "GRCh37": "GRCh37",
+    "grch38": "GRCh38",
+    "GRCh38": "GRCh38",
+}
+
+##### RULES #####
+
+rule _vcf2maf_run:
+    input:
+        vcf = "{out_dir}/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{vcf_name}.vcf",
```

Comment:
This will have to handle bgzipped vcfs.

Reply:
In fact, let's just force the user to provide bgzipped vcfs!

Reply:
okay! Out of curiosity, do we run vcf2maf on the augmented vcf, whic is not bgzipped (from manta module) or just the direct output, which is already bgzipped?

Reply:
Neither. This would likely never be run on Manta outputs. It's for simple somatic mutations such as the output of strelka and varscan.

Reply:
oh okayy should we then wait for strelka/varscan to be finalized before testing?

Reply:
Almost certainly. If you put Varscan in a separate PR I can help finalize that one but given what I've learned thus far, it will need some changes

Reply:
I have varscan in a separate PR already, but currently it relies on vcf2maf. I can remove vcf2maf from varscan's PR

Reply:
sure